sci-mathematics/netgen/netgen-5.3.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools eutils flag-o-matic multilib versionator
+
+MY_PN=${PN}-mesher
+MY_PV=$(get_version_component_range 1-2)
+DESCRIPTION="NETGEN is an automatic 3d tetrahedral mesh generator"
+HOMEPAGE="http://www.hpfem.jku.at/netgen/"
+SRC_URI="mirror://sourceforge/project/${MY_PN}/${MY_PN}/${MY_PV}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="LGPL-2.1"
+KEYWORDS="~amd64 ~x86"
+IUSE="-ffmpeg jpeg -mpi opencascade"
```

Comment:
I think openmp is used automatically see below the error message at compilation failure you've got -fopenmp:

`In file included from togl_1_7.h:16:0,
                 from ngpkg.cpp:28:
/usr/include/togl-1.7/togl_ws.h:5:0: warning: "TOGL_X11" redefined
 # define TOGL_X11
 ^
<command-line>:0:0: note: this is the location of the previous definition
/bin/sh ../libtool  --tag=CXX   --mode=link mpic++  -march=native -O2 -pipe -ggdb -mavx -mmmx -msse -msse2 -msse3 -mssse3 -msse4.1 -msse4.2 -mcx16 -mpopcnt -fopenmp -DPARALLEL -export-dynamic -Wl,-O1 -Wl,--as-needed -L/usr/lib64/opencascade-6.9.0/ros/lin/lin/lib64 -L/usr/lib64/openmpi/ -o netgen demoview.o ngappinit.o onetcl.o parallelfunc.o ngpkg.o ../libsrc/visualization/libvisual.la ../libsrc/csg/libcsgvis.la ../libsrc/csg/libcsg.la ../libsrc/interface/libinterface.la ../libsrc/meshing/libmesh.la -L/usr/lib64/Togl1.7 -L/usr/lib64/Togl1.7 -lTogl1.7 -lGLU -ltk8.5 -ltcl8.5 -L/usr/local/lib -lmetis  -ljpeg -lGL -lXmu -lX11   
libtool: link: mpic++ -march=native -O2 -pipe -ggdb -mavx -mmmx -msse -msse2 -msse3 -mssse3 -msse4.1 -msse4.2 -mcx16 -mpopcnt -fopenmp -DPARALLEL -Wl,-O1 -o .libs/netgen demoview.o ngappinit.o onetcl.o parallelfunc.o ngpkg.o -Wl,--export-dynamic  -Wl,--as-needed -L/usr/lib64/opencascade-6.9.0/ros/lin/lin/lib64 -L/usr/lib64/openmpi/ ../libsrc/visualization/.libs/libvisual.a ../libsrc/csg/.libs/libcsgvis.so /var/tmp/portage/sci-mathematics/netgen-5.3.1/work/netgen-5.3.1/libsrc/csg/.libs/libcsg.so ../libsrc/csg/.libs/libcsg.so ../libsrc/interface/.libs/libinterface.so /var/tmp/portage/sci-mathematics/netgen-5.3.1/work/netgen-5.3.1/libsrc/meshing/.libs/libmesh.so ../libsrc/meshing/.libs/libmesh.so -lz /usr/lib64/libmpi_cxx.so -L/usr/lib64 /usr/lib64/libmpi.so /usr/lib64/libopen-rte.so /usr/lib64/libopen-pal.so -ldl -lnsl -lutil /usr/lib/gcc/x86_64-pc-linux-gnu/4.9.3/libgomp.so -L/usr/lib64/Togl1.7 -lTogl1.7 -lGLU -ltk8.5 -ltcl8.5 -L/usr/local/lib /usr/lib64/libmetis.so -lm -ljpeg -lGL -lXmu -lX11 -pthread -fopenmp
/var/tmp/portage/sci-mathematics/netgen-5.3.1/work/netgen-5.3.1/libsrc/meshing/.libs/libmesh.so: undefined reference to `netgen::Mesh::ParallelMetis(netgen::Array<int, 0, int>&, netgen::Array<int, 0, int>&, netgen::Array<int, 0, int>&)'
collect2: error: ld returned 1 exit status
`